atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -439,6 +439,13 @@ def get_operator_manifests(workflow, fallback=NO_FALLBACK):
     return get_value(workflow, 'operator_manifests', fallback)
 
 
+def get_image_size_limitation(workflow):
+    config = get_value(workflow, 'image_size_limitation', {})
+    return {
+        'regular_image': config.get('regular_image', 0),
+    }
```

Comment:
Should be renamed to `limit` as well

Reply:
@chmeliik Fixed this and other variable names having "limitation". Thanks!